src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -32,10 +33,9 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
         public bool IncludePrivateMembers { get; private set; }
 
         /// <summary>
-        /// If not empty then compiler should enable instrumentation.
-        /// The string specifies instrumentation settings.
+        /// Type of instrumentation that should be added to the output binary.
         /// </summary>
-        public string Instrument { get; private set; }
+        public HashSet<string> Instrument { get; private set; }
```

Comment:
This is certainly wrong since HashSet is mutable. I'd prefer not use strings. If we use strings then ImmutableArray<string> would be more appropriate.

---
In reply to: [73979817](https://github.com/dotnet/roslyn/pull/13013#discussion_r73979817) [](ancestors = 73979817)